packages/turbo-workspaces/src/install.ts

```diff
@@ -68,6 +68,18 @@ export const PACKAGE_MANAGERS: Record<
       semver: ">=2",
     },
   ],
+  bun: [
+    {
+      name: "bun1",
+      template: "bun",
+      command: "bun",
+      installArgs: ["install"],
+      version: "1.x",
+      executable: "bunx",
+      semver: "<2",
```

Comment:
I think this is the syntax?
```suggestion
      semver: ">= 1.0.1 && <2",
```

We don't work less than 1.0.1.